src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseHandler.cs

```diff
@@ -120,6 +120,27 @@ public override Task<InitializeViewResult> InitializeObjectView(InitializeViewRe
                             IsAzureDB = isAzureDB
                         };
 
+                        // Collect the Database properties information
+                        if (!requestParams.IsNewObject)
+                        {
+                            var smoDatabase = dataContainer.SqlDialogSubject as Database;
+                            databaseViewInfo.ObjectInfo = new DatabaseInfo()
+                            {
+                                Name = smoDatabase.Name,
+                                CollationName = smoDatabase.Collation,
+                                DateCreated = smoDatabase.CreateDate.ToString(),
+                                LastDatabaseBackup = smoDatabase.LastBackupDate == DateTime.MinValue ? "None" : smoDatabase.LastBackupDate.ToString(),
+                                LastDatabaseLogBackup = smoDatabase.LastLogBackupDate == DateTime.MinValue ? "None" : smoDatabase.LastLogBackupDate.ToString(),
+                                MemoryAllocatedToMemoryOptimizedObjectsInMb = ConvertKbtoMbString(smoDatabase.MemoryAllocatedToMemoryOptimizedObjectsInKB),
+                                MemoryUsedByMemoryOptimizedObjectsInMb = ConvertKbtoMbString(smoDatabase.MemoryUsedByMemoryOptimizedObjectsInKB),
+                                NumberOfUsers = smoDatabase.Users.Count.ToString(),
+                                Owner = smoDatabase.Owner.ToString(),
+                                SizeInMb = smoDatabase.Size.ToString("0.00") + " MB",
```

Comment:
You shouldn't add the units here - that can be added on the UI side as well

(and TBH this should probably be a number, not sure why it would need to be a string)

Reply:
The reason that I'm adding units here is to have all conversions on STS side and to display on ADS UI directly with the prepared values. 

But, I agree with you, adding units on UI side would be good!

Reply:
Since, we are displaying the values as 16.00 MB, so considered it as string here, I can change it to its original type/int and process on UI side.